superset/viz.py

```diff
@@ -203,8 +203,6 @@ def query_obj(self):
 
     @property
     def cache_timeout(self):
-        if self.form_data.get('cache_timeout'):
```

Comment:
Given that the cache is explorer view agnostic the cache timeout is only associated with the query and not the explorer `form_data`.